core/dku_config/README.md

```diff
@@ -23,7 +23,7 @@ You want to validate in backend that the parameters have been filled properly an
 message. You can then build a DkuConfig object.
 
 ```python
-from dkulib.dku_config.dku_config import DkuConfig
+from core.dku_config.dku_config import DkuConfig
```

Comment:
This is a bit closer to how it would be used in a plugin
```suggestion
from dkulib.core.dku_config.dku_config import DkuConfig
```

Reply:
Hmm though then I suggest we also add the command for how to add it as a git submodule named "dkulib" if you agree?

Reply:
Good idea. Let's document this in the main README at the root of the repository

Reply:
Alright, added a usage section to the main README